ironfish/src/wallet/account.ts

```diff
@@ -259,19 +269,23 @@ export class Account {
         'Existing asset owner should match public address',
       )
 
-      const existingSupply = existingAsset.supply
-      const supply = existingSupply - value
+      let supply = existingAsset.supply
+      if (blockHash && sequence) {
```

Comment:
If there's no blockHash and sequence we won't update anything on the asset. Should we return early?